docs/user/QoRDashboard.md

```diff
@@ -0,0 +1,5 @@
+# Using the OpenROAD Dashboard to track and analyze Quality of Results
+
+```{warning}
+This page is under construction. Check back later!
+```
```

Comment:
@habibayassin
@luarss is working on our documentation, and we would like to include the great work you have done with the new dashboard. Could you help Jack write a small description of your work and create a user guide to showcase the currently deployed features?

Reply:
@vvbandeira yes of course... @luarss I will send you an email with these details.